speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/CHANGES.txt

```diff
@@ -49,16 +49,12 @@ Added:
             DataSelectionMenu.xml
             DeveloperMenu.xml
             EndExperimentMenu.xml
-            raceresultsmenu.xml
```

Comment:
should indeed not be there, but should be in the changed list (see line 400 for xml's) with what has been changed